pkg/steps/cluster_claim.go

```diff
@@ -116,14 +116,17 @@ func (s *clusterClaimStep) run(ctx context.Context) error {
 	if err := s.hiveClient.Create(ctx, claim); err != nil {
 		return fmt.Errorf("failed to created cluster claim %s in namespace %s: %w", claim.Name, claim.Namespace, err)
 	}
-	logrus.WithField("claim.Name", claim.Name).WithField("claim.Namespace", claim.Namespace).Info("waiting for the claimed cluster to be ready ...")
+	logrus.Info("Waiting for the claimed cluster to be ready.")
 	claim = &hivev1.ClusterClaim{}
 	if err := wait.Poll(15*time.Second, s.clusterClaim.Timeout.Duration, func() (bool, error) {
```

Comment:
This should really be a watch.

Reply:
I will do with another PR since this one is about logging.

Reply:
https://github.com/openshift/ci-tools/pull/1874